src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -2793,7 +2706,7 @@ Real64 GLHEVert::calcPipeConvectionResistance(EnergyPlusData &state)
     Real64 nusseltNum = 0.0;
     if (reynoldsNum < lower_limit) {
         nusseltNum = 4.01; // laminar mean(4.36, 3.66)
-    } else if (lower_limit <= reynoldsNum && reynoldsNum < upper_limit) {
+    } else if (reynoldsNum < upper_limit) {
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:2796]:(style),[knownConditionTrueFalse],Condition 'lower_limit<=reynoldsNum' is always true